src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1163,6 +1163,11 @@
     "create index fact_values_value_float_idx on fact_values(value_float)"
     "create index fact_values_value_integer_idx on fact_values(value_integer)"))
 
+(defn add-corrective-change-index
+  []
+  (jdbc/do-commands
+    "CREATE INDEX resource_events_status_for_corrective_change_idx ON resource_events (status) WHERE corrective_change;"))
```

Comment:
So the goal is to create a new index for 'status', that only maps corrective_change = true cases? So that lookup is faster? Wasn't sure if the index on status was intentional or not, or if you mean corrective_change.

Its no issue if this is true, but you should probably explain the 'why' or specifics like this that might confuse the reviewer in your commit message a little better, so I don't need to ask the question :-).